posts/2024-07-03_introducing_a_ne.../introducing_a_new_coursera_course_for_hands_on_clinical_data_science_using__r..qmd

```diff
@@ -0,0 +1,85 @@
+---
+title: "Introducing a [new Coursera course](https://www.coursera.org/learn/hands-on-clinical-reporting-using-r) for hands-on clinical data science using R."
+author:
+  - name: "Joel Laxamana & Stefan Thoma"
+description: |
+  "Releasing [our second (hands-on) Coursera course](https://www.coursera.org/learn/hands-on-clinical-reporting-using-r), aimed at enhancing the skills of data scientists and shedding light on the impact of R open-source software within the industry."
+date: "2024-07-03"
+categories: [Community]
+image: "pharmaverse.png"
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "2024-07-03_introducing_a_ne..."
+# renv::use(lockfile = "renv.lock")
+```
+
+<!--------------- post begins here ----------------->
+
+```{r, echo = FALSE}
+link::auto()
+```
+
+Shifting to R in an industry traditionally reliant on SAS is no small feat.
+It requires not just new tools, but also significant upskilling for programmers.
+Roche-Genentech is addressing this need by offering three publicly available Coursera courses aimed at enhancing the skills of current data scientists within the pharmaceutical sector and introducing our work to those outside the industry.
+In this post, we are excited to announce the release of [our second Coursera course](https://www.coursera.org/learn/hands-on-clinical-reporting-using-r), and we will discuss how R open-source software is being embraced across the industry and particularly within Roche/Genentech.
```

Comment:
```suggestion
In this post, we are excited to announce the release of [our second Coursera course](https://www.coursera.org/learn/hands-on-clinical-reporting-using-r), and we will discuss how R open-source software is being embraced across the industry and particularly within Roche-Genentech.
```

Reply:
Hi, @StefanThoma .  This was a minor update for consistency across the blog.  Some show 'Roche/Genentech' and others could be 'Roche-Genentech', could we fix and just select one of these and ensure all are consistent?